gem2rpm/smart_proxy_plugin.spec.erb

```diff
@@ -112,8 +112,12 @@ mv %{buildroot}%{gem_instdir}/bundler.d/%{plugin_name}.rb \
 
 # sample config
 mkdir -p %{buildroot}%{foreman_proxy_settingsd_dir}
-mv %{buildroot}%{gem_instdir}/settings.d/%{plugin_name}.yml.example \
+<% config_files = spec.files.select { |f| f.start_with?('config/') } +
+                  spec.files.select { |f| f.start_with?('settings.d/') } %>
+<% config_files.each do |config_file| %>
+mv %{buildroot}%{gem_instdir}/<%= config_file %> \
    %{buildroot}%{foreman_proxy_settingsd_dir}/%{plugin_name}.yml
```

Comment:
> It also bundles any .yml files found in these folders. In case the settings file name doesn't match with the plugin name, this will fix it.

It's only going to keep one file and not multiple, because they're all being moved to the same destination filename. I think it might be best to keep the loop for every file, but chomp any .example suffixes from the filename and copy it exactly into place.

It's probably best to preserve the original filename instead of using %{plugin_name} for two reasons:

1. It supports multiple files, if a plugin had them.
2. The smart proxy doesn't just load every file, it loads specific filenames based on the internal plugin name or the plugin configuration. The plugin author knows best, so %{plugin_name} may be incorrect.